src/core/importType.js

```diff
@@ -109,20 +109,23 @@ function isExternalLookingName(name) {
   return isModule(name) || isScoped(name);
 }
 
-function typeTest(name, context, path ) {
+function typeTest(name, context, getPath) {
   const { settings } = context;
   if (isInternalRegexMatch(name, settings)) { return 'internal'; }
-  if (isAbsolute(name, settings, path)) { return 'absolute'; }
+  if (isAbsolute(name, settings)) { return 'absolute'; }
```

Comment:
i very intentionally had all of these pass all three arguments, for consistency. if we're changing that, then this only takes 1 argument.
```suggestion
  if (isAbsolute(name)) { return 'absolute'; }
```

(similar feedback on all the others)